docker/tests/os.Dockerfile

```diff
@@ -103,18 +143,64 @@ RUN set -euxv; \
               vim \
               # nm for lwhich
               binutils \
-              git; \
+              git \
+              # For lfs.zip
+              unzip; \
       fi; \
     fi
 
 SHELL ["/usr/bin/env", "bash", "-euxvc"]
 
+COPY --from=conda-python /usr/local /opt/python
 COPY --from=docker /usr/local /usr/local
 COPY --from=docker-compose /usr/local /usr/local
 COPY --from=docker-compose_musl /usr/local/bin/docker-compose /usr/local/bin/docker-compose_musl
 COPY --from=docker-compose_glib /usr/local/bin/docker-compose /usr/local/bin/docker-compose_glib
+COPY --from=git-lfs /usr/local /usr/local
+COPY --from=jq /usr/local /usr/local
 RUN shopt -s nullglob; for patch in /usr/local/share/just/container_build_patch/*; do "${patch}"; done
 
+# TODO: I want to (use just install functions? and) "fix" docker-compose using conda,
+# in a container_build_patch, instead here, so that would mean conda only gets
+# added to the image in that case, right now its there 100% of the time, and only
+# used for a few images.
+RUN if ! docker-compose --version; then \
+      rm /usr/local/bin/docker-compose; \
+      # Handle symlink dirs, grrr
+      cd /opt/python; \
+      for d in *; do \
```

Comment:
can't you use a follow symlink flag or something?

Reply:
I don't believe that will fix the problem